app/renderer/js/components/webview.js

```diff
@@ -117,6 +117,11 @@ class WebView extends BaseComponent {
 	}
 
 	getBadgeCount(title) {
+		const mutedOrganizations = ConfigUtil.getConfigItem('mutedOrganizations');
+		const { url } = this.props;
+		if (!!mutedOrganizations[url]) {
```

Comment:
Jus out of curiosity, any reason to use `!!mutedOrganizations[url]` instead of simply `mutedOrganizations[url]` ?

Reply:
Not really, just the way I got used to. I'll change this along with the Travis fixes and squash the commits.